t/create-withdrawal-data.2-of-3-nonsegwit.golden

```diff
@@ -36,8 +36,8 @@ Is this data correct?
 *** WARNING: Incorrect data may lead to loss of funds ***
 
 0.20000000 BTC in unspent supplied transactions
-0.19996310 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
 0 BTC going back to cold storage address 2NGPJX8kzdRpAQJuZWMpCBth1umjQFHeFcz
+0.19996310 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
```

Comment:
I don't understand why this should change with a python version bump?

Reply:
This is the same issue from 60e189b4883cbe6ea43e2304a02157372d53c764, the ordering of the outputs changed for some of the withdrawals. It was effectively random before; now it is consistent and predictable.